packages/conf-adwaita-icon-theme/conf-adwaita-icon-theme.1/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+maintainer: "7895506+MSoegtropIMC@users.noreply.github.com"
+homepage: "https://github.com/adwaita/adwaita-icon-theme"
+bug-reports: "https://gitlab.adwaita.org/adwaita/adwaita-icon-theme/issues"
+authors: "GNU Project"
+license: ["LGPL-3.0-only" "CC-BY-SA-3.0"]
+# E.g. on Fedora the system package does not contain a .pc file.
+# Testing for this folder seems to be more portable than using system packager commands as fallback.
+build: [ "sh" "-c" "pkg-config --short-errors --print-errors adwaita-icon-theme || [ -d /usr/share/icons/Adwaita ]"]
+depends: [
+  "conf-pkg-config" {build}
+]
+depexts: [
+  # The list of supported os-family names can be derived from the opam sources
+  #   https://github.com/ocaml/opam/blob/6aefe95e60084e63d01b1c7c028b6b77de1f839f/src/state/opamSysInteract.ml#L84-L123
+  # The package name for many Linux/BSD/macOS/cygwin/msys platforms can be searched here
+  #   https://repology.org specifically https://repology.org/project/adwaita-icon-theme/versions
+  #   Not listed above but supported by opam are suse, oraclelinix, ol, rhel
+  [ "adwaita-icon-theme" "adwaita-icon-theme-dev" ] {os-family = "alpine"}
+  [ "adwaita-icon-theme" ] {os-family = "amzn"}
+  [ "adwaita-icon-theme" ] {os-family = "arch"}
+  [ "adwaita-icon-theme" ] {os-family = "archlinux"}
```

Comment:
Do you need both these lines? Isn't `["gnome-icon-theme"] {os-distribution = "arch"}` enough?

Reply:
I am not sure. I got the OS-family values systematically from the opam sources, and this specific one from:

(https://github.com/ocaml/opam/blob/6aefe95e60084e63d01b1c7c028b6b77de1f839f/src/state/opamSysInteract.ml#L108)

But maybe I should look for the back translation of `type families` - I am not sure how this works.